api/controllers/users.js

```diff
@@ -0,0 +1,61 @@
+const User = require("../models/user");
+const bcrypt = require("bcrypt");
+
+const UsersController = {
+  Create: async (req, res) => {
+    console.log("CREATING USER");
+
+    const hashedPassword = await bcrypt.hash(req.body.password, 10);
+
+    const user = new User({
+      userName: req.body.userName,
+      password: hashedPassword,
+      email: req.body.email,
+      bio: req.body.bio,
+    });
+
+    await user.save((err) => {
+      if (err) {
+        res.status(400).json({ message: "Bad request" });
```

Comment:
Instead of saying 'Bad Request' for the message.
You could link the err to the message:
res.status(400).json({ message: err });